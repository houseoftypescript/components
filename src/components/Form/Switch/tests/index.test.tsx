import { render } from '@testing-library/react';
import { Switch } from '../index';
import React from 'react';

describe('Switch', () => {
  it('to match snapshot', () => {
    const { container } = render(<Switch />);
    expect(container).toMatchSnapshot();
  });
});
