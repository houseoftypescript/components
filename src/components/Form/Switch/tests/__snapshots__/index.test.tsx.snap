// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch to match snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe hGtDRL"
  >
    <div
      class="sc-gswNZR fgUEz"
    />
  </div>
</div>
`;
